File: Selection Sort/selection_sort.py
def SelectionSort(arr):
    for i in range(len(arr)):
        k = i
        for j in range(i + 1, len(arr)):
            if arr[k] > arr[j]:
                k = j
        arr[i], arr[k] = arr[k], arr[i]
    return arr

File: Selection Sort/selection_sort.js
const SelectionSort = (arr) => {
  for (let i = 0; i < arr.length; ++i) {
    let k = i;
    for (let j = i + 1; j < arr.length; ++j) if (arr[k] > arr[j]) k = j;
    [arr[i], arr[k]] = [arr[k], arr[i]];
  }
  return arr;
};
